src/main/webapp/app/pages/oncologyTherapiesPage/oncologyTherapiesPage.tsx

```diff
@@ -0,0 +1,679 @@
+import React, { useEffect, useMemo, useState } from 'react';
```

Comment:
This file is based on deleted precisionOncologyTx page. But there are too many changes, you have to review all unfortunately. @calvinlu3